src/wp-includes/class-wp-admin-bar.php

```diff
@@ -19,21 +19,13 @@ class WP_Admin_Bar {
 	public $user;
 
 	/**
-	 * @since 3.3.0
+	 * Deprecated menu property.
 	 *
-	 * @param string $name
-	 * @return string|array|void
+	 * @since 3.1.0
+	 * @deprecated 3.3.0 Modify admin bar nodes with WP_Admin_Bar::get_node(), WP_Admin_Bar::add_node(), and WP_Admin_Bar::remove_node().
```

Comment:
```suggestion
	 * @deprecated 3.3.0 Modify admin bar nodes with WP_Admin_Bar::get_node(),
 	 *                   WP_Admin_Bar::add_node(), and WP_Admin_Bar::remove_node().
```
Wrapping per the documentation standards. DocBlocks should not extend beyond a total of 120 characters wide.